io/disk/fiotest.py

```diff
@@ -139,7 +140,7 @@ def setUp(self):
                         self.cancel("PMEM engines not built with fio")
 
         if self.disk:
-            if self.disk not in disk.get_disks():
+            if self.disk not in disk.get_all_disk_paths():
```

Comment:
@abdhaleegit we have identified some recent changes on these patches.
can you please look into this PR and accommodate into these PRs.
https://github.com/avocado-framework-tests/avocado-misc-tests/pull/2612

Reply:
@Naresh-ibm already on top of those changes,, have my code rebase to above patch already

Reply:
btw,  fio test is not in the PR you shared